amun/overlays/cnv-prod/amun-api/imagestreamtag.yaml

```diff
@@ -39,3 +39,28 @@ spec:
     importPolicy: {}
     referencePolicy:
       type: Local
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: workflow-helpers
+spec:
+  tags:
+  - name: latest
+    from:
+      kind: DockerImage
+      name: quay.io/thoth-station/workflow-helpers:latest
```

Comment:
we should not use the `latest` tag as we do not know what is that from here, but have a specific tag here for our components for easily debug and make sure that we have the version we want.

Same for messaging below.